### ISSUE TITLE
Height range modifier does not work

### ISSUE THREAD

**Issue by SirRenix** [supermerill/SuperSlicer#4023]
### What happened?

When customizing skirt mods, I noticed that the infill option does not work. The percent and pattern setting does not apply to the current christmas edition.  And the pattern is shifting. Pictures attached.

### Project file & How to reproduce

Old Version
![image](https://github.com/supermerill/SuperSlicer/assets/7069603/f5b4e300-900c-4392-8f95-a4d10b746735)
Aktuelle Christmas Version
![image](https://github.com/supermerill/SuperSlicer/assets/7069603/21cf171e-a230-45fe-9839-9686c8d7056f)

### Version

2.5.59.4

### Operating system

Windows 11

### Printer model

Voron 2.4 Stealthburner CANBus

**Comment by supermerill** [comment]
yes, that's why I added the disclaimer 'known issue : modifiers don't work' to the release

**Comment by supermerill** [comment]
same source as #4016
